### PR TITLE
Add style to desktop version of portfolio website

### PR DESCRIPTION
**Pull request by Newtayo**
- Updated the images on the homepage and about me page.
- Responsive functionality has been added to site to enable go view on mobile and desktop device

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -220,10 +336,12 @@ <h2 class="aboutheader">About me</h2>
...
-
+        </div>
           
         </ul>
       </div>
+    </div>
+     
     </section>
 
     <section class="contact-form">
```
Comment (wayungi) [inline index.html:352]: - [ ] You have done very good work implementing most of the project features 👏🏾 . We just have a little change required here to fully match the Figma design. For your implementation, use the [Google fonts](https://fonts.google.com/). The font you need is called Inter

-[ ] Your paragraph's alignment is does not match the figma design. Please left align it.

_**Figma**_
text-1

_**Your design**_
text-2

File: index.html [hunk 2]
```diff
@@ -220,10 +336,12 @@ <h2 class="aboutheader">About me</h2>
             <li class="list">Gitlab</li>
             <li class="list">Terminal</li>
           </ul>
-
+        </div>
```
Comment (wayungi) [inline index.html:344]: - [ ] Good work implementing your languages, skills, and Frameworks. To fully match the Figma, we need to reduce the spacing between the language list items. check the images below for comparison.

_**Figma**_
lang-1

_**Your design**_
lang-2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Newtayo  👋🏾  

You have done a very impressive job so far! 🌟 
There are some issues that you still need to work on 🔨 🔧  to go to the next project but you are almost there!
image

## Highlights
- [ ] Good PR title ✔️ 
- [ ] GitHub flow ✔️ 
- [ ] Descriptive commit message ✔️ 
- [ ] Professional README ✔️ 
- [ ] All linter checks passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Newtayo** [comment]
- Rearranged the text beside the contact form
- Applied font-family "inter" as in the figma design template
- Rearranged the language, skill, and framework sections as recommended in the review

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @Newtayo , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Good design

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**